### PR TITLE
Try disabling running tests in parallel to fix System_UsesProvidedDirectoryAndCertificate 

### PR DESCRIPTION
**Pull request by pakrym**
@pranavkm

### CODE REVIEW

File: test/Microsoft.AspNetCore.DataProtection.Extensions.Test/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+// Workaround for DataProtectionProviderTests.System_UsesProvidedDirectoryAndCertificate
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.DataProtection.Extensions.Test/Properties/AssemblyInfo.cs]: Maybe include the bug id here?

**Comment by pranavkm** [comment]
Does this actually work?

**Comment by pakrym** [comment]
Yes, stops test failure on win7 machine.

**Comment by pranavkm** [comment]
:shipit:

**Pull request merged**